pom.xml

```diff
@@ -267,7 +302,7 @@
         </identity.application.auth.basicauth.imp.pkg.version.range>
 
         <!-- Carbon Kernel version -->
-        <carbon.kernel.version>4.4.5</carbon.kernel.version>
+        <carbon.kernel.version>4.4.11</carbon.kernel.version>
```

Comment:
Any particular reason for updating the kernel version?

Reply:
yes. Since new kernel version handle password as object instead of string